src/filtrering/filtrering-label-container.tsx

```diff
@@ -171,31 +196,31 @@ function FiltreringLabelContainer({
                 ];
             } else if (key === 'foedeland') {
                 return value.map(singleValue => {
-                    if (foedelandListData.get(singleValue) != null) {
-                        return (
-                            <FiltreringLabel
-                                key={`${key}--${singleValue}`}
-                                label={'Fødeland: ' + foedelandListData.get(singleValue)}
-                                slettFilter={() => slettEnkelt(key, singleValue)}
-                            />
-                        );
-                    } else {
-                        return '';
-                    }
+                    return (
+                        <FiltreringLabel
+                            key={`${key}--${singleValue}`}
+                            label={
+                                'Fødeland: ' +
+                                (foedelandListData.has(singleValue) ? foedelandListData.get(singleValue) : 'ugyldig')
```

Comment:
Er 'ugyldig' en situasjon man skal kunne komme i?

Reply:
Det kan skje hvis kodeverk har ikke beskrivelse for land kode. Det skjedde for en språk i en jira saker fra forige uka. Det er bare sikkerhetstiltak.